extensions/wikia/PageHeader/templates/PageHeaderController_categories.php

```diff
@@ -0,0 +1,29 @@
+<?php if ( $categories->hasVisibleCategories() ): ?>
+	<div class="page-header__categories">
+		<span class="page-header__categories-in" data-tracking="categories-top-in"><?= $categories->inCategoriesText ?>:</span>
+		<div class="page-header__categories-links">
+			<?= $categories->visibleCategoriesHTML ?>
```

Comment:
@weronika generally, it's better to generate HTML from a template. That's why I would prefer to leave the dropdown categories below in the template.

As explained in the `Categories.class.php`, we can't easily do the same for the list of visible categories. Each category element has `display: inline-block` and if we have a new line character between the `<a>` and the comma, browser adds a space between them. This can be avoided by using creative templating like:
```
</a><--
-->,
```
but we agreed with @latata that it's not something we want to do.